boards/xtensa/intel_s1000_crb/doc/index.rst

```diff
@@ -84,44 +87,58 @@ Run the installer using these commands:
 
    cd ~/Downloads
    chmod +x Xplorer-6.0.4-linux-installer.bin
-   sudo ./Xplorer-6.0.4-linux-installer.bin
+   ./Xplorer-6.0.4-linux-installer.bin
 
-Xplorer software will be installed into the /opt/xtensa folder. Please note a dialogue box
-should pop-up after running this command. Otherwise, it means your system is missing some
-package which is preventing successful installation, most probably gtk2-i686.  You can
-install this missing package with:
+Please note a dialogue box should pop-up after running this command. In case the
+graphical installation tool does not start, the tool will revert to console
+based installation. The graphical tool is the preferred installation method.
 
-.. code-block:: console
+If the graphical tool does not start, it means your system is missing some
+packages which is preventing successful installation, most probably
+``gtk2-i686``.  You can install any missing packages with::
```

Comment:
The Xtensa installer and tools are 32-bit binaries. It's worth adding instructions to install 32-bit compatibility libraries in a 64-bit Linux development host.